ledger/ledger-security-test-lib/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//bazel_tools:scala.bzl", "da_scala_library")
+
+da_scala_library(
+    name = "ledger-security-test-lib",
```

Comment:
I would name the entire package `ledger-security-evidence` or similar and have both targets defined here in it